tests/pytest/conftest.py

```diff
@@ -44,12 +44,16 @@ def model_PemData():
 @pytest.fixture
 def model_AuthProvider():
     auth_provider = AuthProvider.objects.create(
-        sign_out_button_label="Sign out",
+        sign_out_button_template="core/includes/sign-out-button--senior.html",
+        sign_out_link_template="core/includes/sign-out-link--senior.html",
         client_name="Client",
         client_id="1234",
         authority="https://example.com",
     )
 
+    assert not auth_provider.supports_claims_verification
+    assert auth_provider.supports_sign_out
```

Comment:
What is the reason for having these assertions here rather than in the test?

Reply:
Yeah there wasn't any explicit tests for these models, I could add them instead of asserting here.

Reply:
Oh I see, `supports_sign_out` is not on `EligibilityVerifier` anymore and that's the model we have tests for

Reply:
I'm ok with leaving them here

Reply:
I guess it just felt a little weird writing tests for the fixtures the tests are creating... 😆 

But we already do that, and the tests confirm the behavior in our standard way, so good call.